src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -726,7 +726,6 @@
                                            (schema/required-key "catalog") schema/Bool}
       (schema/optional-key "trusted_facts") {(schema/required-key "values") {schema/Str schema/Any}}
       (schema/optional-key "facts") {(schema/required-key "values") {schema/Str schema/Any}}
-      (schema/optional-key "job_id") schema/Int
```

Comment:
I think we should leave this in, because it will need to be there ultimately.

Reply:
I removed it to reduce confusion with code_id, so when we actually add support for it, it will come in more cleanly in a single PR/commit instead of just dangling unused.